### PR TITLE
Unify and modernize code and layout

### PR DESCRIPTION
**Pull request by baldersheim**
@havardpe PR

### CODE REVIEW

**Comment by baldersheim** [comment]
@havardpe PR
Now this should be reviewable, with size reporting too.

File: eval/src/apps/analyze_onnx_model/analyze_onnx_model.cpp [hunk 1]
```diff
@@ -229,30 +260,34 @@ int probe_types() {
...
     }
+    MemoryUsage vm_before = extract_memory_usage();
     Slime result;
     auto &root = result.setObject();
     auto &types = root.setObject("outputs");
     Onnx model(params["model"].asString().make_string(), Onnx::Optimize::DISABLE);
     Onnx::WirePlanner planner;
-    for (size_t i = 0; i < model.inputs().size(); ++i) {
-        auto spec = params["inputs"][model.inputs()[i].name].asString().make_string();
+    for (const auto & i : model.inputs()) {
```
Comment (havardpe) [inline eval/src/apps/analyze_onnx_model/analyze_onnx_model.cpp:269]: consider better name for i, like model_input

File: storage/src/vespa/storage/distributor/activecopy.cpp [hunk 2]
```diff
@@ -162,16 +162,16 @@ ActiveCopy::calculate(const Node2Index & idealState, const lib::Distribution& di
...
 }
 
 void
 ActiveList::print(std::ostream& out, bool verbose, const std::string& indent) const
 {
     out << "[";
     if (verbose) {
-        for (size_t i=0; i<_v.size(); ++i) {
-            out << "\n" << indent << "  " << _v[i].nodeIndex() << " " << _v[i].getReason();
+        for (auto i : _v) {
```
Comment (havardpe) [inline storage/src/vespa/storage/distributor/activecopy.cpp:173]: consider better name for i

File: eval/src/apps/analyze_onnx_model/analyze_onnx_model.cpp (hunk 1 above)
Comment (baldersheim) [reply eval/src/apps/analyze_onnx_model/analyze_onnx_model.cpp]: Fixed

File: eval/src/apps/analyze_onnx_model/analyze_onnx_model.cpp [hunk 3]
```diff
@@ -47,17 +53,42 @@ void extract(const vespalib::string &str, const vespalib::string &prefix, vespal
...
+    size_t v(0);
+    size_t end = s.find("kB");
+    auto [ptr,ec] = std::from_chars(s.data(), s.data()+std::min(s.size(), end), v, 10);
+    if (ec != std::errc()) {
+        throw std::runtime_error(fmt("Bad format : '%s' at '%s'", s.c_str(), ptr));
+    }
+    if (end != vespalib::string::npos) {
+        return v * 1024;
+    }
+    throw std::runtime_error(fmt("Bad format : %s", s.c_str()));
```
[the remark this answers is not in the document]
Comment (baldersheim) [reply eval/src/apps/analyze_onnx_model/analyze_onnx_model.cpp]: Fixed

File: storage/src/vespa/storage/distributor/activecopy.cpp (hunk 2 above)
Comment (baldersheim) [reply storage/src/vespa/storage/distributor/activecopy.cpp]: Fixed

**Pull request merged**